Resources/Maps/_Goobstation/barratry.yml

```
@@ -76631,6 +76631,13 @@ entities:
     - type: Transform
       pos: 4.5405226,-14.581865
       parent: 1
+- proto: MaterialSilo
+  entities:
+  - uid: 15714
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)